### ISSUE TITLE
[tikv][client] TiConfiguration  doesn't correctly implement serializable 

### ISSUE THREAD

**Issue by leonardBang** [tikv/client-java#474]
### 1. Describe the bug

TiConfiguration  doesn't correctly implement serializable interface

### 2. Minimal reproduce step (Required)
You can reproduce this issue when you serialize a TiConfiguration Object.

### 3. What did you see instead (Required)

The `Optional` is not serializable
```
  private Optional<Integer> rawKVReadSlowLogInMS = getIntOption(TIKV_RAWKV_READ_SLOWLOG_IN_MS);
  private Optional<Integer> rawKVWriteSlowLogInMS = getIntOption(TIKV_RAWKV_WRITE_SLOWLOG_IN_MS);
  private Optional<Integer> rawKVBatchReadSlowLogInMS =
      getIntOption(TIKV_RAWKV_BATCH_READ_SLOWLOG_IN_MS);
  private Optional<Integer> rawKVBatchWriteSlowLogInMS =
      getIntOption(TIKV_RAWKV_BATCH_WRITE_SLOWLOG_IN_MS);
```
### 4. What did you expect to see? (Required)
Use `transient` to fix it.

### 5. What is your Java Client and TiKV version? (Required)

- Client Java: 3.2.0-snapshot
- TiKV:

**Comment by marsishandsome** [comment]
it is fixed in https://github.com/tikv/client-java/pull/445

**Comment by leonardBang** [comment]
@marsishandsome Thanks for the fix, do you have plan to release 3.2 version recently?
